src/web/app/pages-student/student-home-page/__snapshots__/student-home-page.component.spec.ts.snap

```diff
@@ -389,7 +389,18 @@ exports[`StudentHomePageComponent should snap with default fields 1`] = `
   studentFeedbackSessionStatusSubmitted={[Function String]}
   timezoneService={[Function TimezoneService]}
 >
+  <div>
+     No courses you are enrolled in. Please refer to the 
+    <a
+      href="/web/student/help"
+      ng-reflect-router-link="/web/student/help"
+      routerlink="/web/student/help"
+    >
+      Help page
```

Comment:
Any particular reason for capitalising the "H" in "Help"?

Reply:
Done.